test/scripts/update-param-sys.py

```diff
@@ -0,0 +1,97 @@
+########################################################################
+# This script is designed to function with OpenC3 COSMOS's Script Runner
+########################################################################
+from openc3.script import *
+import struct
+
+# Clear counts and data products from prior tests:
+cmd("Linux_Example Ccsds_Command_Depacketizer_Instance-Reset_Counts")
+cmd("Linux_Example Command_Router_Instance-Reset_Data_Products")
```

Comment:
Nice. There are probably other counts you want to reset too, like in the command router.